cmd/edit/ingress/cmd.go

```diff
@@ -118,10 +118,15 @@ func run(cmd *cobra.Command, argv []string) {
 		interactive.Enable()
 	}
 
+	cluster := r.FetchCluster()
 	var labelMatch *string
 	if cmd.Flags().Changed(labelMatchFlag) {
+		if ocm.IsHyperShiftCluster(cluster) {
+			r.Reporter.Errorf("Updating route selectors is not supported for hosted cp clusters")
```

Comment:
Is this already done in clusters service as well?

Reply:
No. Just in rosa

Reply:
I was wrong. It is done indirectly:

```
	if patch.RouteSelectors != nil {
		if ingress.Default {
			err = errors.BadRequest.UserErrorf(
				"Can't add route selectors to default ingress '%s' in cluster '%s'",
				ingress.ID,
				cluster.ID)
			return
		}
		clauses["route_selectors"] = patch.RouteSelectors
	}
```